apps/schools/api_views/boundary.py

```diff
@@ -251,7 +251,7 @@ class Admin3s(KLPListAPIView, CacheMixin):
     def get_queryset(self):
         btype = self.request.GET.get('school_type', 'all')
         meetingreport = self.request.GET.get('meetingreport', None)
-        qset = Boundary.objects.all_active()\
+        qset = Boundary.objects.all_active().filter(id__gte=settings.STATE_STARTING_BOUNDARY_ID)\
             .select_related('boundarycoord__coord', 'parent__hierarchy', 'hierarchy__name')\
             .prefetch_related('parent', 'hierarchy')
 
```

Comment:
All `Boundary.objects.all_active().filter(id__gte=settings.STATE_STARTING_BOUNDARY_ID)` reflects the hack to fetch only the boundaries that belong to Odisha. Should ideally be a `State` filter.